### PR TITLE
Mention limitation with transactional fixtures in README

### LINKED ISSUE #15: Query on slave returning wrong results in test
Hey,

After switching some queries to be sent to the slaves, I've noticed a specific test failing due to the `on_slave` scope returning wrong results:

```
[2] pry(#<RSpec::ExampleGroups::ShopsController::GETIndex>)> Shop.where("created_on > ?", 5.days.ago)
[master]  Shop Load (1.2ms)  SELECT `shops`.* FROM `shops` WHERE (created_on > '2017-01-14 11:54:33.639784')
 # => [#<Shop:0x007fa103931290>]
# returns the `Shop` record as expected, test succeeds

[3] pry(#<RSpec::ExampleGroups::ShopsController::GETIndex>)> Shop.on_slave.where("created_on > ?", 5.days.ago)
[slave]  Shop Load (0.8ms)  SELECT `shops`.* FROM `shops` WHERE (created_on > '2017-01-14 11:56:41.160893')
# => [] 
# test fails
```

Oddly enough, `Slavery.on_slave` still sends the queries to the master:

```
[4] pry(#<RSpec::ExampleGroups::ShopsController::GETIndex>)> Slavery.on_slave { Shop.where("created_on > ?", 5.days.ago) }
[master]  Shop Load (0.9ms)  SELECT `shops`.* FROM `shops` WHERE (created_on > '2017-01-14 11:58:43.070048')
# => [#<Shop:0x007fa103931290>]
```

The failing test is roughly the following:

```ruby
  describe "foo" do
    let!(:shop) { FactoryGirl.create(:shop) }
    
    it "foooooo" do
      shop.update_attribute :created_on, 3.days.ago
      shop.update_attribute :updated_on, 3.days.ago

      get :index, { :since => 5.days.ago }
      expect(assigns(:shops)).to eq([shop])
    end
  end
```

I've also tried reducing the query to `Shop.count`/`Shop.on_slave.count`, same issue: the result is `1` vs. `0`.

Any ideas?

Thanks

### PR DESCRIPTION
**Pull request by agis-**
Fixes #15.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,20 @@ With this line, Slavery stops connection switching and all queries go to the mas
 
 This may be useful when one of the master or the slave goes down. You would rewrite `database.yml` to make all queries go to the surviving database, until you restore or rebuild the failed one.
 
+## Transactional fixtures
+
+When `use_transactional_fixtures` is set to `true`, it's NOT recommended to
+write to the database besides fixtures, since the slave connection is not aware
+of changes performed in the master connection due to [transaction isolation]().
+
```
Comment (kenn) [inline README.md]: Link to wikipedia is missing?

File: spec/active_record/log_subscriber_spec.rb [hunk 2]
```diff
@@ -10,10 +10,11 @@
 
     before do
       ActiveRecord::Base.logger = logger
+      @backup_disabled = Slavery.disabled
```
Comment (kenn) [inline spec/active_record/log_subscriber_spec.rb]: Do we still need this change?

Comment (agis-) [inline spec/active_record/log_subscriber_spec.rb]: This is irrelevant of the issue and should actually be a separate commit outside of this PR. It's always good to restore the value to what it was before, like we already do in other tests.

File: README.md (hunk 1 above)
Comment (agis-) [inline README.md]: Doh! Fixed :)

File: spec/active_record/log_subscriber_spec.rb (hunk 2 above)
Comment (kenn) [inline spec/active_record/log_subscriber_spec.rb]: Oh I missed what's in the `after` block - thanks! 👍

**Comment by kenn** [comment]
Thank you! 🎉 🎉
